testproject/Assets/Tests/Runtime/testproject.runtimetests.asmdef

```diff
@@ -1,11 +1,25 @@
 {
     "name": "TestProject.RuntimeTests",
+    "rootNamespace": "",
     "references": [
         "Unity.Multiplayer.MLAPI.Runtime",
         "Unity.Multiplayer.MLAPI.RuntimeTests",
-        "TestProject.ManualTests"
+        "TestProject.ManualTests",
+        "UnityEngine.TestRunner",
+        "UnityEditor.TestRunner"
     ],
-    "optionalUnityReferences": [
-        "TestAssemblies"
-    ]
+    "includePlatforms": [],
+    "excludePlatforms": [],
+    "allowUnsafeCode": false,
+    "overrideReferences": true,
+    "precompiledReferences": [
+        "nunit.framework.dll"
+    ],
+    "autoReferenced": false,
+    "defineConstraints": [
+        "UNITY_INCLUDE_TESTS",
+        "UNITY_EDITOR"
+    ],
+    "versionDefines": [],
+    "noEngineReferences": false
```

Comment:
and again, diff could just be:

```diff
+    "defineConstraints": [
+        "UNITY_INCLUDE_TESTS",
+        "UNITY_EDITOR"
+    ],
```

Reply:
same comment as above

Reply:
those references are basically expanded version of `TestAssemblies`